incubator/hnc/config/samples/hnc_v1_config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: hnc.x-k8s.io/v1alpha1
+kind: HNCConfiguration
+metadata:
+  name: config
+spec:
+  types:
+    - apiVersion: v1
+      kind: Secret
+      mode: propagate
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: Role
+      mode: ignore
```

Comment:
Oh, can you reverse these two? Role must always be `propagate`, but Secret can be `ignore`.

Reply:
Sure! Done.